app/src/main/java/com/eventyay/organizer/core/faq/list/FaqListFragment.java

```diff
@@ -237,9 +237,11 @@ public void showMessage(String message) {
 
     @Override
     public void showResults(List<Faq> items) {
-        if(items.isEmpty()) {
+        if (items.isEmpty()) {
             showEmptyView(true);
             return;
+        } else {
+            showEmptyView(false);
```

Comment:
Move this out of if condition and change to `showEmptyView(items.isEmpty());`

Reply:
Done.